src/mlpack/methods/decision_tree/random_binary_numeric_split.hpp

```diff
@@ -0,0 +1,124 @@
+/**
```

Comment:
I forgot this until now, but I think we need to add this (and the implementation file) to the list of files in `CMakeLists.txt`. :+1:

Reply:
Thanks. I have added it 👍